### PR TITLE
commit 1

### PR DESCRIPTION
**Pull request by tankxu**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116925207%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116925956%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116926062%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116926292%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116926521%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116929582%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116929845%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r127421136%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/5%23issuecomment-315326469%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23issuecomment-315326469%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22sss%22%2C%20%22created_at%22%3A%20%222017-07-14T10%3A23%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4507629%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/fatjyc%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%203d3057c74bf9528e8d15275671ea835a7c6b251d%20commitcomment%202%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116925207%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22do%20not%20commit%20to%20master%20branch%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A11%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20commitcomment%3AL1-3%22%7D%2C%20%22Pull%20eb1af4b0ff7d7531970e2c3240d35ac604788040%20commitcomment%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116926062%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22delete%20this%20code%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A16%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20commitcomment%3AL1-4%22%7D%2C%20%22Pull%201809a20aced7c51c2dff9354d793f5a3799aee38%20commitcomment%202%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116929582%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22how%20about%20commit%20to%20the%20same%20branch.%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A39%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%2C%20%7B%22body%22%3A%20%22ssdfasdf%22%2C%20%22created_at%22%3A%20%222017-07-14T09%3A39%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4507629%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/fatjyc%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20commitcomment%3AL1-3%22%7D%2C%20%22Pull%20eb1af4b0ff7d7531970e2c3240d35ac604788040%20commitcomment%202%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116929845%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22you%20are%20right%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A41%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20commitcomment%3AL1-4%22%7D%2C%20%22Pull%20eb1af4b0ff7d7531970e2c3240d35ac604788040%20commitcomment%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116925956%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22great%20new%20branch%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A15%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20don%27t%20like%20this%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A19%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20commitcomment%3AL1-4%22%7D%2C%20%22Pull%20eb1af4b0ff7d7531970e2c3240d35ac604788040%20commitcomment%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/5%23discussion_r116926292%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20like%20this%22%2C%20%22created_at%22%3A%20%222017-05-17T07%3A17%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20commitcomment%3AL1-4%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 3d3057c74bf9528e8d15275671ea835a7c6b251d commitcomment 2'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#discussion_r116925207'>File: commitcomment:L1-3</a></b>
- <a href='https://github.com/tankxu'></a> do not commit to master branch
- [ ] <a href='#crh-comment-Pull eb1af4b0ff7d7531970e2c3240d35ac604788040 commitcomment 3'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#discussion_r116925956'>File: commitcomment:L1-4</a></b>
- <a href='https://github.com/tankxu'></a> great new branch
- <a href='https://github.com/tankxu'></a> I don't like this
- [ ] <a href='#crh-comment-Pull eb1af4b0ff7d7531970e2c3240d35ac604788040 commitcomment 4'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#discussion_r116926062'>File: commitcomment:L1-4</a></b>
- <a href='https://github.com/tankxu'></a> delete this code
- [ ] <a href='#crh-comment-Pull eb1af4b0ff7d7531970e2c3240d35ac604788040 commitcomment 1'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#discussion_r116926292'>File: commitcomment:L1-4</a></b>
- <a href='https://github.com/tankxu'></a> I like this
- [ ] <a href='#crh-comment-Pull 1809a20aced7c51c2dff9354d793f5a3799aee38 commitcomment 2'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#discussion_r116929582'>File: commitcomment:L1-3</a></b>
- <a href='https://github.com/tankxu'></a> how about commit to the same branch.
- <a href='https://github.com/fatjyc'></a> ssdfasdf
- [ ] <a href='#crh-comment-Pull eb1af4b0ff7d7531970e2c3240d35ac604788040 commitcomment 2'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#discussion_r116929845'>File: commitcomment:L1-4</a></b>
- <a href='https://github.com/tankxu'></a> you are right
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/5#issuecomment-315326469'>General Comment</a></b>
- <a href='https://github.com/fatjyc'></a> sss

<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/5?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/5?approve=1'></a>&nbsp;<a href='https://github.com/tankxu/test-ci/pull/5'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by tankxu** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/tankxu/test-ci/pull/5'>Refresh</a>.<a href='#crh-update-none'></a>

File: commitcomment [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hello world
+commit directly to the master branch.
```
Comment (tankxu) [inline commitcomment]: do not commit to master branch

File: commitcomment [hunk 2]
```diff
@@ -1,2 +1,3 @@
 hello world
-commit directly to the master branch.
+commit directly to the new branch.
```
Comment (tankxu) [inline commitcomment]: great new branch

File: commitcomment [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
+Create a new branch
```
Comment (tankxu) [inline commitcomment]: delete this code

File: commitcomment [hunk 4]
```diff
@@ -0,0 +1,3 @@
+hello world
```
Comment (tankxu) [inline commitcomment:1]: I like this

File: commitcomment [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+commit directly to the new branch.
+Create a new branch
```
Comment (tankxu) [inline commitcomment]: I don't like this

File: commitcomment [hunk 6]
```diff
@@ -1,3 +1,2 @@
 hello world
 commit directly to the new branch.
```
Comment (tankxu) [inline commitcomment]: how about commit to the same branch.

File: commitcomment [hunk 7]
```diff
@@ -1,2 +1,3 @@
 hello world
-commit directly to the master branch.
```
Comment (tankxu) [inline commitcomment]: you are right

File: commitcomment (hunk 6 above)
Comment (fatjyc) [reply commitcomment]: ssdfasdf

**Comment by fatjyc** [comment]
sss

**Pull request closed without merging**